README.md

```diff
@@ -62,10 +116,28 @@ Assume variables x, y, and z are numbers.
 **a)**
 Write an expression for the mean (i.e. average) of x, y, and z.
 
+<pre>
+I did both below. 
+</pre>
 **b)**
 Write a series of expressions to adjust each of x, y, and z halfway toward their mean.
 That is, reset the value of each variable to something based on its previous value.
 
+<pre> 
+function problem2b() {
+
+    var x = 3;
+    var y = 4;
+    var z = 5;
+
+    var m = (x+y+z) / 3;
+
+    x += (m-x)/2;
+    y += (m-y)/2;
+    z += (m-z)/2;
+
+};
```

Comment:
Nice work! The only thing I'd note is that you can make your code a bit more readable if you use, for example, `mean` instead of `m`.